src/Sylius/Behat/Client/RequestBuilder.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Client;
+
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class RequestBuilder
+{
+    private string $uri;
+    private string $method;
+    private array $parameters = [];
+    private array $headers = [];
+    private array $content = [];
+    private array $files = [];
+
+    private function __construct(string $uri, string $method)
+    {
+        $this->uri = $uri;
+        $this->method = $method;
+    }
+
+    public static function create(string $uri, string $method): self
+    {
+        return new self($uri, $method);
+    }
+
+    public function withHeader(string $name, string $value): self
+    {
+        $this->headers[$name] = $value;
+
+        return $this;
+    }
+
+    public function withFile(string $key, UploadedFile $file): self
+    {
+        $this->files[$key] = $file;
+
+        return $this;
+    }
+
+    public function withContent(array $content): self
+    {
+        $this->content = $content;
+
+        return $this;
+    }
+
+    public function withPartialContent(string $key, array $content): self
+    {
+        $this->content[$key] = $content;
+
+        return $this;
+    }
+
+    public function withoutPartialContent(string $key, array $content): self
+    {
+        $this->content[$key] = $content;
+
+        return $this;
+    }
```

Comment:
What is the difference between these two functions?

Reply:
Yea, I deleted it as they are not used yet.